src/main/java/org/kpmp/geneExpressionSummary/SNRNAGeneExpressionSummaryRepository.java

```diff
@@ -24,6 +24,9 @@
 	List<SNRNAGeneExpressionExpressionSummaryValue> findExpressionSummaryPerGeneByCellTypeAndTissueType(
 			@Param("cellType") String cellType, @Param("tissueType") String tissueType);
 
+	@Query(value = "SELECT COUNT(*) FROM sn_rnaseq snr WHERE snr.gene= :geneSymbol AND snr.tissue_type= :tissueType", nativeQuery = true)

```

Comment:
I can't remember (and the card doesn't say) if we wanted these counts to be filtered by the gene searched for. If that is the case, then, ignore this comment.